speak.md

```diff
@@ -5,22 +5,99 @@ permalink: /speak/
 header-img: img/headers/hobart-audrey.jpg
 ---
 
-We want you to speak. 
+## [Click here to submit a talk proposal now!](https://www.papercall.io/pyconau2018)
+
+We want you [to present at PyCon AU](https://www.papercall.io/pyconau2018)!
 
 Tell us about what you've been hacking on, what drives you, what excites you. What have you been learning? What do you want to share?
 
-The Call for Presentations will open in early 2018, so start thinking. 
+The 2018 Call for Presentations (CFP) is open from Monday, April 23 [Anywhere On Earth (AoE)](https://en.wikipedia.org/wiki/Anywhere_on_Earth) to Monday, May 28 AoE.
 
 Do you have a talk for the main conference track, or something more specialised for our Friday specialist tracks?
 
-The sort of things we're looking for: 
+[Submit now!](https://www.papercall.io/pyconau2018)
+
+If you've never presented at a conference before and think you might like to try it, [submit now!](https://www.papercall.io/pyconau2018)
+
+We're looking for 30 minute talks on everything Python, open source, and tech-related.
+
+We welcome submissions from everybody, including those:
+
+* who have never given a conference talk before,
+* who are new to Python,
+* who have built interesting things in Python, 
+* who have broken interesting things with Python,
+* who have helped build Python, and
+* who have expertise to share with our community from fields outside of Python (and outside of technology).
+
+Thinking of something cool? We want to hear from you!
+
+You can submit a proposal for the main conference, and all of our four Specialist Tracks, [through PaperCall](https://www.papercall.io/pyconau2018) or reach us any time with questions at **<mailto:program@pycon-au.org>**.
+
+## Specialist Tracks
+
+The first day of PyCon AU is dedicated to specialist tracks. These tracks, also known as 'mini-confs', are curated by smaller specialist groups within Python community, and allow for more in-depth talks on the subject. 
+
+In 2018, these tracks will be: DjangoCon, Security and Privacy, Education, and Internet of Things. [Read more](http://2018.pycon-au.org/news/specialist-tracks).
```

Comment:
I personally prefer not to use phrase like "click here" an "read more" as they tend to be less accessibility friendly. How about linking the names of the conferences? `[DjangoCon, Security and Privacy, Education, and Internet of Things](...)`.